main.go

```diff
@@ -85,7 +85,8 @@ func main() {
 	}
 
 	ctrlOptions := ctrl.Options{
-		Scheme: scheme,
```

Comment:
There is an attribute on like 68 called "metrics-addr'. Can we drop it now?

Reply:
Done